parser/prism/Translator.cc

```diff
@@ -406,6 +406,13 @@ std::unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::DString>(parser.translateLocation(loc), std::move(sorbetParts));
         }
+        case PM_IT_LOCAL_VARIABLE_READ_NODE: {
+            [[fallthrough]];
```

Comment:
The implicit fallthrough in cases is often unintended, to the point where compilers have a `-Wimplicit-fallthrough` flag to ban it. C++17 adds this [`fallthrough` attribute](https://en.cppreference.com/w/cpp/language/attributes/fallthrough), as a way to express an intentional fallthrough, and disable that warning.

We don't have `-Wimplicit-fallthrough` enabled, but I like the clarity of the explicit fallthrough.